pkg/scheduler/framework/plugins/nodevolumelimits/non_csi.go

```diff
@@ -481,6 +522,8 @@ func getMaxVolumeFunc(filterName string) func(node *v1.Node) int {
 			return defaultMaxGCEPDVolumes
 		case azureDiskVolumeFilterType:
 			return defaultMaxAzureDiskVolumes
+		case cinderVolumeFilterType:
```

Comment:
Can you open up an issue to investigate if this logic can be removed? I believe we should be always taking the limit from CSINode now.

Reply:
Opened https://github.com/kubernetes/kubernetes/issues/113885